core-rust/core-api-server/src/core_api/handlers/stream_transactions.rs

```diff
@@ -106,7 +105,7 @@ pub fn to_api_committed_transaction(
     receipt: LedgerTransactionReceipt,
     identifiers: CommittedTransactionIdentifiers,
 ) -> Result<models::CommittedTransaction, MappingError> {
-    let receipt = to_api_receipt(context, receipt)?;
+    let receipt = to_api_receipt(context, receipt).unwrap();
```

Comment:
what is the reason for this change?

Reply:
Ah good catch, I had this unwrap for debugging reasons. Will remove.